server/api/index.js

```diff
@@ -0,0 +1,27 @@
+const { default: Axios } = require("axios");
+
+class API {
```

Comment:
Curious whether to use a class with static, or just an module exports a few functions

Reply:
I'll be glad to implement any way you feel is better.